packages/@okta/vuepress-site/docs/reference/scim/scim-11/index.md

```diff
@@ -0,0 +1,672 @@
+---
+title: SCIM 1.1 Protocol Reference
+icon: /assets/img/icons/scim.svg
+meta:
+  - name: description
+    content: Your SCIM API must support specific SCIM 1.1 API endpoints to work with Okta. Those endpoints and their explanations are detailed here.
+---
+
+# Okta and SCIM 1.1 Protocol
+
+This reference focuses on how Okta API endpoints share information with SCIM Version 1.1 specific API calls.
+
+The SCIM Protocol is an application-level, REST protocol for provisioning and managing identity data on the web. The protocol supports creation, discovery, retrieval, and modification of core identity Resources.
+
+## User operations
+
+### Creating users
+
+<ApiOperation method="post" url="/users" />
+
+The user creation operation brings the user’s application profile from Okta over to the service provider. An user's application profile represents the key-value pairs defined in the Profile tab when a user is added.
+
+To enable user provisioning, an Okta administrator must configure the provisioning options under the Okta Admin Console. For more information on enabling the provisioning features of your SCIM application, see [Provisioning and Deprovisioning](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Provisioning_Deprovisioning_Overview), under the **Accessing Provisioning Features** section.
+
+After saving the application configuration, Okta runs two requests:
+
+    GET /scim/v1/users?startIndex=1&count=2 HTTP/1.1
```

Comment:
We don't indent for code, we use triple backticks with a syntax highlight language specified.